poseval/common/transformations.py

```diff
@@ -133,19 +183,55 @@ def _raw_board_to_sparse_representation(self, piece_to_layer_map, piece_to_value
                 values.append(piece_to_value_map[c])
                 column = column + 1
 
-        return ({'indices_z': indices_z, 'indices_x': indices_x, 'indices_y': indices_y, 'values': values})
+        return {'indices_z': indices_z, 'indices_x': indices_x, 'indices_y': indices_y, 'values': values}
 
     def _raw_board_to_flat_vector(self, piece_to_layer_map, piece_to_value_map, layers):
         raw_board = self.raw_board()
-        row = 0; column = 0
-        vector = [0 for _ in range(0, 8*8*layers)]
+        row = 0
+        column = 0
+        vector = [0 for _ in range(0, 8 * 8 * layers)]
```

Comment:
How about
```suggestion
        vector = [0] * (8 * 8 * layers)
```

Reply:
It was the other guy's code, your change didn't break tests so I implemented it ;p